portfolio/src/main/webapp/index.html

```diff
@@ -25,10 +25,20 @@ <h1>Tal's Portfolio</h1>
       <div id="fact-container"></div>
  
       <p>Post a comment here!</p>
+      
+      
       <div id = 'notLoggedIn'>
           <p>Login To Comment</p>
-          <button target="/user">Login</button>
+          <button target="loginUrl">Login</button>
```

Comment:
This button doesn't seem to be doing anything when I click it. It should be redirecting me to login